GotExplorer.BLL/Services/Results/ServiceResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentValidation.Results;
+
+namespace GotExplorer.BLL.Services.Results
+{
+    public class ServiceResult<T>
+    {
+        public T? ResultObject { get; set; }
+        public ValidationResult? ValidationResult { get; set; } = null;
```

Comment:
You do not need this wrapper. You need just to use ValidationResult and other native classes from fluent validation library